docs/source/datasets/index.rst

```diff
@@ -26,6 +26,9 @@ A **mailing address** represents the address a simulant uses to receive mail,
 which may be different -- for example, a PO box.
 Mailing addresses, not physical addresses, are recorded in tax filings.
 
+Some columns in the generated data may contain missing values, even if no noise has been added to 
+the data. These missing values are represented by :code:`numpy.nan`.
```

Comment:
```suggestion
Some fields are not applicable to every record in a simulated dataset,
so some columns may contain "missing" values, even if no noise has been added to the data.
For example, most addresses do not have a unit number, and some do not have a street number, so the :code:`unit_number` and/or :code:`street_number` fields will be "missing" for many rows in any dataset that contains addresses.
Similarly, columns pertaining to spouse or dependents in the 1040 tax dataset are not applicable to every simulant, so these columns also contain missing values.
Values that are missing because they are not applicable are represented by :code:`numpy.nan`.
```
I think we should explain **why** fields are missing in the unnoised data and give some examples, so I've tried to do that here. I think reason for "valid missingness" as we were calling it, is that some columns are _not applicable_ to every row, so that's how I've phrased it. Does that sound right? Feel free to make edits if you think the wording could be better.